lib/honeybadger/logger.ex

```diff
@@ -65,6 +67,19 @@ defmodule Honeybadger.Logger do
 
   ## Helpers
 
+  defp notify(reason, metadata, stacktrace) do
```

Comment:
This is doing the same work as `Honeybadger.notify/3` (including overwriting the breadcrumb metadata). Can it be extracted somewhere to make it easier to test and prevent subtle differences?

Reply:
I agree they are doing some similar operations, but they seemed different enough that I wasn't sure unifying them would be worth it. Subtle things like the fact that the lookup keys are different, one returns a different output, they store different breadcrumbs, and one makes sure a key is removed.

I'm open to suggestions if it seems like there is a way to simplify.